capture-top-level-project/gradle-enterprise-custom-user-data.groovy

```diff
@@ -0,0 +1,6 @@
+// This script is meant to be used with the common-custom-user-data-maven-extension.
+// It needs to be placed in .mvn/gradle-enterprise-custom-user-data.groovy in the root project directory.
+// (see https://docs.gradle.com/enterprise/maven-extension/#using_the_common_custom_user_data_maven_extension)
+
+buildScan.value('topLevelProject.name', session.topLevelProject.name)
```

Comment:
⭕️ Okay, this (maven) is really confusing... to the topLevelProject is the project the build was started with although there might be an enclosing project (the one with the `.mvn` folder). I wonder whether `topLevelProject` is - although technically correct - a good name for this custom value.

Reply:
I'm also not sure about the name. I decided to stick with Maven's name for this (which is also inconsistent internally since some places call it "execution root") instead of coming up with a new name.

@etiennestuder WDYT?

Reply:
Wouldn't the "execution root" be the place where you've executed `mvn` - which is not necessarily the project root, if you only build a submodule.